control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -164,6 +170,19 @@ type HostedControlPlaneReconciler struct {
 	awsSession                    *session.Session
 	reconcileInfrastructureStatus func(ctx context.Context, hcp *hyperv1.HostedControlPlane) (InfrastructureStatus, error)
 	NameServerIP                  string
+
+	KubeClient        kubernetes.Interface
+	OperatorClient    v1helpers.StaticPodOperatorClient
```

Comment:
We will remove this when https://github.com/openshift/library-go/pull/1593 merges.